crt_portal/cts_forms/templates/forms/phone_pro_template.html

```diff
@@ -32,6 +32,19 @@ <h1 class="intake-section-title padding-bottom-2m margin-left-5">{{title}}</h1>
 
       {% csrf_token %}
       <h2>Contact</h2>
+ 
+      <!-- 
+        We should probably clean up all of the "if section == ..." stuff 
+       as this is going to become a nightmare in terms of readability
+       and therefore difficult to maintain. I would suggest we split each section's
+       form out into its own template, but that also gets messy because we then are duplicating
+       templates... so I don't really have a good solution at this time. I will think about this
+       more when I have time as right now I am focused on delivering an MVP for the ELS-CRU form.
+       -->
```

Comment:
Remove comment

I'm okay with this "if section", because it's very specific to the section and prevents duplication of the entire form template